sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -718,6 +718,11 @@ public partial class BlobServiceStatistics
         internal BlobServiceStatistics() { }
         public Azure.Storage.Blobs.Models.BlobGeoReplication GeoReplication { get { throw null; } }
     }
+    public enum BlobSetExpiryRelativeTo
```

Comment:
This doesn't fit very well with the .NET Framework Design Guidelines naming conventions.  Maybe `BlobExpirationOffset` or `BlobExpirationTarget` or something similar?

Reply:
Renamed to ```BlobExpirationOffset```, and ```DataLakeFileExpirationOffset```